src/modules/Dropdown/Dropdown.js

```diff
@@ -547,7 +547,6 @@ export default class Dropdown extends Component {
     } else {
       this.setValue(value)
       this.handleChange(e, value)
-      this.close()
```

Comment:
This was the bug, we were closing every time an item was activated.  Even when `closeOnBlur` was false.  Since, this method was called on blur due to blur activating the selected item.